client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -73,9 +73,14 @@ export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
 				</div>
 			</div>
 			{ isDesktopView && ! isDesktopApp && (
-				<Button className="go-mobile__email-link-button" onClick={ emailLogin }>
-					{ translate( 'Email download link' ) }
-				</Button>
+				<div className="go-mobile__email-link">
+					{ translate(
+						'We can email a link to open on your phone to instantly download the app.'
+					) }
+					<Button className="go-mobile__email-link-button" onClick={ emailLogin } borderless>
+						{ translate( 'Email download link' ) }
+					</Button>
```

Comment:
🤔 Why not just make this a link?

Reply:
I was using the convention we have (maybe loosely) that "anchors are for navigation, button is meant for interactions" (or something to that effect). It pops up in the linter if I try changing it to an `<a>` tag.

Reply:
If that's the case, we'd probably keep it as a button with button styling, right? Users will expect links to navigate by default.

Reply:
In terms of accessibility, anchors without an `href` should be also avoided: https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/mouse-and-keyboard-events/links/link-behavior/

In regards to using a button with a link styling, just noting we already have some instances in My Home, like when skipping a site setup step:

<img width="1064" alt="Screen Shot 2020-05-07 at 09 34 45" src="https://user-images.githubusercontent.com/1233880/81267005-0db07500-9046-11ea-9295-1c2b176e41b2.png">

Reply:
> In regards to using a button with a link styling, just noting we already have some instances in My Home, like when skipping a site setup step

I think those are intended to be a `borderless` button. Could the mobile card button be a borderless button, too, or does that add unnecessary padding around it?